src/protocol/taiko_alethia/TaikoInbox.sol

```diff
@@ -26,34 +28,44 @@ contract TaikoInbox {
         maxAnchorBlockIdOffset = _maxAnchorBlockIdOffset;
     }
 
-    function publish(uint256 nBlobs, bytes calldata data, uint64 anchorBlockId) external {
+    function publish(uint256 nBlobs, uint64 anchorBlockId) external {
```

Comment:
This is not an issue using our canonical data feed. However, if a user comes with a custom implementation, there might be side effects they must be aware of

```suggestion
    /// WARNING: Make sure `datafeed` is trusted given it can reenter when calling `publish` while the state
    /// is inconsistent (i.e. `prevPublicationId` has not been set)
    function publish(uint256 nBlobs, uint64 anchorBlockId) external {
```

Reply:
Same for the `delayedInclusionStore`